app/src/main/res/layout/contact_detail_list_items.xml

```diff
@@ -0,0 +1,653 @@
+                android:layout_centerVertical="true"
+                android:layout_marginEnd="10dp"
+                android:layout_marginRight="10dp"
+                android:src="@drawable/ic_fax_black_24dp"
+                android:tint="@color/colorAccent" />
+
+            <TextView
+                android:id="@+id/textFax"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginLeft="20dp"
+                android:layout_marginStart="20dp"
+                android:layout_toEndOf="@id/imageFax"
+                android:layout_toRightOf="@id/imageFax"
+                android:text="+91 123 321321"
+                android:textAppearance="@android:style/TextAppearance.Medium" />
+
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@id/textFax"
+                android:layout_marginLeft="20dp"
+                android:layout_marginStart="20dp"
+                android:layout_marginTop="8dp"
+                android:layout_toEndOf="@id/imageFax"
+                android:layout_toRightOf="@id/imageFax"
+                android:text="Fax"
+                android:textAppearance="@android:style/TextAppearance.Small" />
+
+        </RelativeLayout>
+    </LinearLayout>
+
+    <LinearLayout
+        android:id="@+id/editFaxLayout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:visibility="gone">
+
+        <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:padding="8dp">
+
+            <ImageView
+                android:id="@+id/editImageFax"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_centerVertical="true"
+                android:layout_marginEnd="10dp"
+                android:layout_marginRight="10dp"
+                android:src="@drawable/ic_fax_black_24dp"
+                android:tint="@color/colorAccent" />
+
+            <EditText
+                android:id="@+id/editFax"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginLeft="20dp"
+                android:layout_marginStart="20dp"
+                android:layout_toEndOf="@id/editImageFax"
+                android:layout_toRightOf="@id/editImageFax"
+                android:hint="Fax"
+                android:inputType="number"
+                android:singleLine="true"
+                android:textAppearance="@android:style/TextAppearance.Medium"
+                android:textColorHint="#607D8B" />
+
+        </RelativeLayout>
+    </LinearLayout>
+</LinearLayout>
```

Comment:
Here you have created different layout for edit and view only. Just make two LinearLayout with editableLayouts and readonlyLayouts and put all controls in that layouts. So its easy to just toggle two view instead each of the control layout.